src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -1057,7 +1057,12 @@ async function processDataType(
 function prepareNumber(a, dataType) {
   return {
     size: a.$.size,
-    is_signed: a.$.name.endsWith('u') || !a.$.name.includes('int') ? 0 : 1,
+    is_signed:
+      a.$.name.endsWith('u') ||
+      !a.$.name.includes('int') ||
+      a.$.name.includes('endpoint')
```

Comment:
Why this hardcoding for "endpoint"?  What about other numeric types that have interesting names?  Why is "endpoint" special?  at bare minimum, this needs to be documented.

Reply:
got rid of the hardcoding